recipes/conversation_view/message_input/message_input.vue

```diff
@@ -191,19 +213,30 @@
                 @focusin="sendButtonFocus = true"
                 @focusout="sendButtonFocus = false"
               >
-                <template #icon>
+                <template
+                  v-if="!isEdit"
+                  #icon
+                >
                   <dt-icon
                     name="send"
                     size="300"
                   />
                 </template>
+                <template
+                  v-if="isEdit"
+                >
+                  <p>{{ 'Save Changes' }}</p>
+                </template>
```

Comment:
Did we want to center middle the tooltip in this case now that it is wider?

![Screenshot 2023-11-15 at 3 29 03 PM](https://github.com/dialpad/dialtone-vue/assets/64808812/7cd4cdd9-a3a2-422a-89c0-807a9e68b76d)

Reply:
when its edit mode, the tooltip will read save changes, so it still will look wider with that tooltip, hence kept the position same.